python/cdp/openapi_client/api/evm_accounts_api.py

```diff
@@ -3262,7 +3261,7 @@ async def update_evm_account(
         self,
         address: Annotated[str, Field(strict=True, description="The 0x-prefixed address of the EVM account. The address does not need to be checksummed.")],
         x_idempotency_key: Annotated[Optional[Annotated[str, Field(min_length=36, strict=True, max_length=36)]], Field(description="An optional [UUID v4](https://www.uuidgenerator.net/version4) request header for making requests safely retryable. When included, duplicate requests with the same key will return identical responses.  Refer to our [Idempotency docs](https://docs.cdp.coinbase.com/api-v2/docs/idempotency) for more information on using idempotency keys. ")] = None,
-        update_evm_account_request: Optional[UpdateEvmAccountRequest] = None,
+        create_evm_account_request: Optional[CreateEvmAccountRequest] = None,
```

Comment:
weird that the generated EVM client no longer has UpdateEvmAccountRequest, but the Solana one still does...